example/lib/pages/network_tile_provider.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:latlong2/latlong.dart';
+
+import '../widgets/drawer.dart';
+
+class NetworkTileProviderPage extends StatelessWidget {
+  static const String route = 'NetworkTileProvider';
+
+  @override
+  Widget build(BuildContext context) {
+    var markers = <Marker>[
+      Marker(
+        width: 80.0,
+        height: 80.0,
+        point: LatLng(51.5, -0.09),
+        builder: (ctx) => Container(
+          child: FlutterLogo(
+            textColor: Colors.blue,
+            key: ObjectKey(Colors.blue),
+          ),
+        ),
+      ),
+      Marker(
+        width: 80.0,
+        height: 80.0,
+        point: LatLng(53.3498, -6.2603),
+        builder: (ctx) => Container(
+          child: FlutterLogo(
+            textColor: Colors.green,
+            key: ObjectKey(Colors.green),
+          ),
+        ),
+      ),
+      Marker(
+        width: 80.0,
+        height: 80.0,
+        point: LatLng(48.8566, 2.3522),
+        builder: (ctx) => Container(
+          child: FlutterLogo(
+            textColor: Colors.purple,
+            key: ObjectKey(Colors.purple),
+          ),
+        ),
+      ),
+    ];
+
+    return Scaffold(
+      appBar: AppBar(title: Text('NetworkTileProvider')),
+      drawer: buildDrawer(context, route),
+      body: Padding(
+        padding: EdgeInsets.all(8.0),
+        child: Column(
+          children: [
+            Padding(
+              padding: EdgeInsets.only(top: 8.0, bottom: 8.0),
+              child: Wrap(children: [
+                Text('This is a map that is using NetworkTileProvider.'),
+                Text('Turn on flight mode to see that images are cached.'),
```

Comment:
We should remove this line, right? This doesn't seem to be true by documentation, right? If that's true, we should rename the provider also.

I think I missed part of the discussion in #894, so just ignore that comment if what I'm saying is wrong....

Reply:
Based on the comments in the code here: https://github.com/fleaflet/flutter_map/blob/issues/829-nullsafety/lib/src/layer/tile_layer.dart#L116

I would say you are correct, and that the opposite is true - it's not cached. Adding the word "not" in the example would probably be enough?

Reply:
@andreandersson Is the `CachedNetworkTileProvider` still a thing, i.e. does it still exist? If not, we should also remove that comment in the code. Thanks for spotting that! :)